### ISSUE TITLE
Touchevent listeners prevent propagation even in static mode

### ISSUE THREAD

**Issue by bulbil** [digirati-co-uk/iiif-canvas-panel#232]
![Screenshot 2024-01-04 at 3 30 39 PM](https://github.com/digirati-co-uk/iiif-canvas-panel/assets/2319626/3031c757-78a7-45fc-8bdc-97965078dd0c)

**Observed Behavior**

On iOS Safari (for example), canvas panel instances using both the responsive and static presets still capture the touch event. In other words, scrolling is prevented if the touch gesture starts within the canvas panel instance.

**Expected Behavior**

Being able to scroll using touch without the event listener capturing the event. On desktop, the same canvas panel examples allows the user to scroll through when using the responsive or static presets, even if the event originates within the canvas panel instance.

**Additional Notes**

I don't know if this is or isn't part of the broader discussion I see mentioned about event models in Canvas Panel.

I have an ulterior motive which is we are trying to implement the "cooperative gesture" pattern used by Google Maps (with related implementations in Mapbox and Leaflet). I would love to be able to do this through Canvas Panel rather than trying to run interference through a synthetic parent element.

Example implementations:

- [Mapbox](https://github.com/mapbox/mapbox-gl-js/pull/11116/files#diff-cd10ae570e4a92c653e31a04dad4c94181fb8ee63f12b2260a11eaf8bc81f3f0)
- [Leaflet plugin](https://github.com/elmarquis/Leaflet.GestureHandling/)

**Comment by stephenwf** [comment]
To answer the first question, about the static preset, you should be able to do this through CSS on the canvas panel component with `pointer-events: none`. However, I understand the "cooperative gesture" issue. 

The Leaflet implementation is more integrated into the APIs that leaflet provide, something that canvas panel tries to abstract (there is a way to get an instance of the internal viewer). An implementation like that would have to be done in the viewer itself.

The Mapbox works by overlaying a "<div />" across the viewer, with the messaging and then hiding and removing the pointer events when you have the modifier key pressed. This is something you could compose with canvas panel, wrapping the whole viewer and having the overlay with the behaviour described.

The "static" preset should probably remove the pointer events and allow you to scroll though, so that could be considered a bug.

**Comment by bulbil** [comment]
Thanks for looking into this so quickly -- I did look into the overlay approach but for whatever reason, I was unable to get a mapbox style overlay with dynamic adding/removing pointer-events to work. I tried everything I could think of. I was wondering whether it had to do with the use of the shadow DOM as that affects how events are propagated.

You mentioned internal viewer -- I did try `cp.withAtlas( r => r.stopControllers() )` and other helpers I found but mobile still did not work as expected.
